search/index.json

```diff
@@ -501,7 +501,7 @@
     "keywords": ["","default","directory","is","that","the","uses?","what","zap"],
     "tags": null,
```

Comment:
I don't remember seeing this before, is that going to be an issue?

Reply:
Er, where did that come from??

Reply:
<shrug> 🤷

Reply:
That's normal.

Reply:
Thanks!